sernet.gs.ui.rcp.main/src/sernet/gs/ui/rcp/main/bsi/views/TemplateView.java

```diff
@@ -0,0 +1,474 @@
+/*******************************************************************************  
+ * Copyright (c) 2016 Viktor Schmidt.  
+ *  
+ * This program is free software: you can redistribute it and/or   
+ * modify it under the terms of the GNU Lesser General Public License   
+ * as published by the Free Software Foundation, either version 3   
+ * of the License, or (at your option) any later version.  
+ * This program is distributed in the hope that it will be useful,      
+ * but WITHOUT ANY WARRANTY; without even the implied warranty   
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.    
+ * See the GNU Lesser General Public License for more details.  
+ *  
+ * You should have received a copy of the GNU Lesser General Public License  
+ * along with this program.   
+ * If not, see <http://www.gnu.org/licenses/>.  
+ *   
+ * Contributors:  
+ *     Viktor Schmidt <vschmidt[at]ckc[dot]de> - initial API and implementation  
+ ******************************************************************************/
```

Comment:
Please create new package _sernet.verinice.rcp.templates_. Move all new RCP classes to this package.